projects/packages/classic-theme-helper/src/class-classic-theme-helper.php

```diff
@@ -43,4 +46,67 @@ public function load_modules() {
 			require_once __DIR__ . $module;
 		}
 	}
+
+	/**
+	 * Conditionally require the Tonesque lib depending on theme support.
+	 */
+	public static function jetpack_load_theme_tools() {
+		if ( current_theme_supports( 'tonesque' ) ) {
+			require_once __DIR__ . '../_inc/lib/tonesque.php';
+		}
+	}
+
+	/**
+	 * Load theme compat file if it exists.
+	 */
+	public static function jetpack_load_theme_compat() {
+
+		/**
+		 * Filter theme compat files.
+		 *
+		 * Themes can add their own compat files here if they like. For example:
+		 *
+		 * add_filter( 'jetpack_theme_compat_files', 'mytheme_jetpack_compat_file' );
+		 * function mytheme_jetpack_compat_file( $files ) {
+		 *     $files['mytheme'] = locate_template( 'jetpack-compat.php' );
+		 *     return $files;
+		 * }
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param array Associative array of theme compat files to load.
+		 */
+		$compat_files = apply_filters(
```

Comment:
This is another place I made changes as the location of the compact files is different.